.docksal/commands/init

```diff
@@ -34,7 +34,11 @@ echo "Waiting 10s for MySQL to initialize...";
 sleep 10
 
 # Site initialization
+echo
 echo -e "${green_bg} Step 2 ${NC}${green} Initializing site...${NC}"
+# This is a one-time command to run only at init
+echo "Downloading Backdrop Drush Commands..."
+fin exec sudo git clone https://github.com/backdrop-contrib/drush.git ~/.drush/backdrop > /dev/null
```

Comment:
@shelane I'm going to suggest that we take this out only because it will be added as of this commit https://github.com/docksal/service-cli/pull/90/commits/f1e46365b0cd4b35dcaed4244b7cc8de2ebb5180